package.json

```diff
@@ -49,8 +49,6 @@
     "lru-cache": "^5.0.0",
```

Comment:
i don't see axios being included in this file? was it already added?

Reply:
it is already there (was added in previous PRs)